src/style.css

```
@@ -1 +1,100 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 450;
+  
+}
+
+body {
+
+  margin: 0;
+
+}
+
+ul {
+
+  margin: 0;
+  
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.image {
+
+  display: flex;
+  align-items: center;
+}
```

no empty line here. fix it in the rest of your code

```suggestion
.image {
  display: flex;
  align-items: center;
}
```